src/qt/languages/de-DE.po

```diff
@@ -1818,6 +1818,29 @@ msgstr "Übertragungsgeschwindigkeit"
 msgid "EMS mode"
 msgstr "EMS-Modus"
 
+msgid "IBM PC/XT 32K Memory Expansion Option"
+msgstr "IBM PC/XT 32K Speichererweiterungsoption"
+
+msgid "IBM PC/XT 64K Memory Expansion Option"
+msgstr "IBM PC/XT 64K Speichererweiterungsoption"
+
+msgid "IBM PC/XT 64/256K Memory Expansion Option"
+msgstr "IBM PC/XT 64/256K Speichererweiterungsoption"
+
+msgid "Generic PC/XT Memory Expansion"
+msgstr "Generische PC/XT Speichererweiterung"
+
+msgid "Microsoft RAMCard for IBM PC"
+msgstr "Microsoft RAMCard für IBM PC"
+
+msgid "IBM PC/AT 128 KB Memory Expansion Option"
+msgstr "IBM PC/AT 128 KB Speichererweiterungsoption"
+
+msgid "Generic PC/AT Memory Expansion"
+msgstr "Generische PC/AT Speichererweiterung"
+
+msgid "
```

Comment:
This stray line breaks compilation, remove it.

Reply:
Yeah sorry, I made some mistakes. Should be fixed now.